genrules.cpp

```diff
@@ -2726,10 +2726,8 @@ int Game::GenRules(const AString &rules, const AString &css, const AString &intr
 		  << "structure which allows transportation of items.  The structures which allow this are: ";
 		last = -1;
 		comma = 0;
-		int j = 0;
```

Comment:
this has not been used, error latest compiler

Reply:
Thanks for catching this.